backend/typescript/models/camp.model.ts

```diff
@@ -35,134 +35,139 @@ export interface Camp extends Document {
+    },
+    campCoordinators: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "User",
+        },
+      ],
+      default: [],
+    },
+    campCounsellors: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "User",
+        },
+      ],
+      default: [],
+    },
+    campSessions: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "CampSession",
+        },
+      ],
+      default: [],
+    },
+    description: {
+      type: String,
+    },
+    dropoffFee: {
+      type: Number,
+      required: true,
+    },
+    pickupFee: {
+      type: Number,
+      required: true,
+    },
+    endTime: {
+      type: String,
+      required: true,
+    },
+    earlyDropoff: {
+      type: String,
+    },
+    fee: {
+      type: Number,
+      required: true,
+    },
+    fileName: {
+      type: String,
+    },
+    formQuestions: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "FormQuestion",
+        },
+      ],
+      default: [],
+      required: true,
+    },
+    latePickup: {
+      type: String,
+    },
+    location: {
+      streetAddress1: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-  },
-  campCounsellors: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "User",
+      streetAddress2: {
+        type: String,
+        required: false,
       },
-    ],
-    default: [],
-  },
-  campSessions: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "CampSession",
+      city: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-  },
-  description: {
-    type: String,
-  },
-  dropoffFee: {
-    type: Number,
-    required: true,
-  },
-  pickupFee: {
-    type: Number,
-    required: true,
-  },
-  endTime: {
-    type: String,
-    required: true,
-  },
-  earlyDropoff: {
-    type: String,
-  },
-  fee: {
-    type: Number,
-    required: true,
-  },
-  fileName: {
-    type: String,
-  },
-  formQuestions: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "FormQuestion",
+      province: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-    required: true,
-  },
-  latePickup: {
-    type: String,
-  },
-  location: {
-    streetAddress1: {
+      postalCode: {
+        type: String,
+        required: true,
+      },
+    },
+    name: {
       type: String,
       required: true,
     },
-    streetAddress2: {
+    campProductId: {
       type: String,
-      required: false,
     },
-    city: {
+    dropoffPriceId: {
       type: String,
-      required: true,
     },
-    province: {
+    dropoffProductId: {
+      type: String,
+    },
+    pickupPriceId: {
+      type: String,
+    },
+    pickupProductId: {
       type: String,
-      required: true,
     },
-    postalCode: {
+    startTime: {
```

Comment:
I'm a little confused about what you actually changed in the model. Could you just summarize the changes in the thread since the additions and removals don't seem to match up

Reply:
Sorry all I added was the `createdAt` and `updatedAt` fields, I think the rest of the fields might be from merges or linting fixes - I'll double check

Reply:
oh ok, thanks

Reply:
Yeah looks like it's all the same except createdAt, updatedAt, and the timestamps field